UI/obs-app.cpp

```diff
@@ -3176,6 +3217,47 @@ static void upgrade_settings(void)
 	os_closedir(dir);
 }
 
+static const char *sentinelContents = "This file intentionally left blank.";
+
+static void check_safe_mode_sentinel(void)
+{
+#ifndef NDEBUG
```

Comment:
I don't think we set `NDEBUG` on any platform but Windows - it is common to explicitly set `DEBUG` when it is enabled, but not set anything otherwise.